ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -637,3 +640,38 @@ def test_paddle_take_along_axis(
         indices=indices,
         axis=axis,
     )
+
+
+# rot90
+@handle_frontend_test(
+    fn_tree="paddle.rot90",
+    dtype_m_k_axes=_get_dtype_values_k_axes_for_rot90(
+        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
Make sure the updated dtypes are included in thte tests too

Reply:
Yup will do!!